ClubManagement/ClubManagement/Fragments/DashboardFragment.cs

```diff
@@ -84,7 +85,7 @@ protected override void DisplayData(string data)
                 tvBudget.Visibility = ViewStates.Visible;
             }
 
-            if (upcomingEvents.Count != 0)
+            if (upcomingEvents != null && upcomingEvents.Any())
```

Comment:
It's not correct. For example, we have 1 event displays on the list and then someone deletes it from another device, so we need to clear the list as well. Please check how other tab keeps the data and applies.